lib/diagnostic.rb

```diff
@@ -135,11 +159,12 @@ def kick
   def multiplier
     case (moves)
     when ['punch', 'move left', 'kick']
-      1.5
+      @damage = 1.5
     when ['kick', 'punch', 'up']
-      2
+      @damage = 2
     else
-      1
+      @damage = 1
     end
   end
+
```

Comment:
The way to make the method chaining work is for `punch`, `kick`, and `move` to return `self`.That way, `ComboAttack.new.punch.move('left').kick.damage` is equivalent to

```ruby
combo = ComboAttack.new
combo.punch
combo.move('left')
combo.kick
combo.damage
```